packages/nice_parser/nice_parser.1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+authors: ["Steffen Smolka <smolka@cs.cornell.edu>"]
+bug-reports: "https://github.com/smolkaj/ocaml-parsing/issues"
+homepage: "https://github.com/smolkaj/ocaml-parsing"
+doc: "https://smolkaj.github.io/ocaml-parsing/nice_parser/"
+license: "MIT"
+dev-repo: "git+https://github.com/smolkaj/ocaml-parsing.git"
+synopsis: "Nice parsers without the boilerplate"
+description:
+  "Nice_parser wraps your {menhir, ocamlyacc}-generated parser in a sane interface, eliminating boilerplate code."
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "dune" {>= "1.10"}
+  "menhir" {build}
+  "stdio" {!= "0"}
+  "base" {!= "0"}
```

Comment:
```suggestion
  "base"
```
What does the `!= 0` means? There is no version 0 for those packages.

Reply:
```suggestion
  "base" {with-test}
```
Oh nevermind it's not even used outside of the tests (modulo what I said above about tests)

Reply:
The opam file was auto generated by dune, and dune insists on a constraint...so I put `!= 0` as a placeholder.